provisioning/tasks/init-debian.yml

```diff
@@ -55,15 +55,6 @@
   with_items:
     - php-common
 
-- name: Purge PHP modules directory.
-  file:
-    path: "{{ item }}"
-    state: absent
-  when: php_purge.changed
-  with_items:
-    - "/usr/lib/php5/modules"
-    - "/usr/lib/php/modules"
-
```

Comment:
Actually, I think we need to test that memcached/redis extensions still work on version change without this. Unfortunately I wont have time to do that until monday.